patterns/dswp-icon-with-excerpt.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Title: DSWP Icon with Excerpt
```

Comment:
Using the word "excerpt" might cause confusion since excerpt has its own meaning in WordPress and is its own kind of block that isn't actually used in this pattern.